content/en/docs/ambient/upgrade/helm/index.md

```diff
@@ -214,7 +214,7 @@ $ helm upgrade istio-ingress istio/gateway -n istio-ingress
 If you have upgraded all data plane components to use the new revision of the Istio control plane, and are satisfied that you do not need to roll back, you can remove the previous revision of the control plane by running:
 
 {{< text syntax=bash snip_id=none >}}
```

Comment:
You don't need to manually keep the tests and the docs in sync - make this

```suggestion
{{< text syntax=bash snip_id=delete_old_revision >}}
```

and this will create a `bash` function with the same name you can call in `revision_test.sh`, so the docs and the tests stay in sync.